src/TLuaInterpreter.cpp

```diff
@@ -3657,61 +3657,37 @@ int TLuaInterpreter::getMudletInfo(lua_State* L)
 {
     Host& host = getHostFromLua(L);
 
-    QByteArrayList knownEncodings{"\"ASCII\""};
-    auto adjustEncoding = [](auto encodingName) {
-        auto originalEncoding = encodingName;
-        if (encodingName.startsWith("M_")) {
-            encodingName.remove(0, 2);
+    QStringList knownEncodings{"ASCII"};
+    QString currentEncoding{host.mTelnet.getEncoding()};
+    {
```

Comment:
Why the `{` ... `}`?

Reply:
Helps scope things, because this function will be huge, handling all sorts of different responsibilities... a trick from you!

Though thinking about it, we could organise everything nicely internally with lambda functions 😏